app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
+        if distance_from_city_center <= 0:
+            raise ValueError(
+                "Distance from city center must be greater than 0"
+            )
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        result = (car.comfort_class * (self.clean_power - car.clean_mark)
+                  * self.average_rating / self.distance_from_city_center)
```

Comment:
The formula for calculating the washing price is correct, but ensure that `distance_from_city_center` is never zero to prevent division by zero errors. This is already handled in the constructor, so no changes are needed here.